docs/plans/2026-03-30_split-mode-phase-1.md

```
@@ -0,0 +1,28 @@
+# Split Mode Phase 1
+
+## Current Repo Check
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)